### DISCUSSION TITLE
[@types/jwplayer] missing getCues()

### DISCUSSION THREAD

**Discussion by mdaoust-sidlee** [DefinitelyTyped/DefinitelyTyped#63409] [Issues with a @types package] [answered]
As seen on the the documentation, jwplayer has the getCues method 
https://docs.jwplayer.com/players/reference/getcues

It should define it in the type definitions here : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jwplayer/index.d.ts 

Thanks in advance.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63737. Please review

**Reply by mdaoust-sidlee** [reply]
Fixed 🎉. Thanks a lot.
